pkg/kubectl/cmd/apply.go

```diff
@@ -135,6 +161,11 @@ func RunApply(f cmdutil.Factory, cmd *cobra.Command, out io.Writer, options *App
 		return err
 	}
 
+	options.pruneResources, err = parsePruneResources(cmdutil.GetFlagStringArray(cmd, "groupversionkind"))
```

Comment:
I think `--all` and `--gourpversionkind` should be allowed to appear together? As the former indicates all objects regardless to label selector and the latter indicates the resource type?